test/parallel/test-buffer.js

```diff
@@ -1108,16 +1108,20 @@ assert.throws(function () {
   new SlowBuffer(smalloc.kMaxLength + 1);
 }, RangeError);
 
-// Test truncation after decode
-var crypto = require('crypto');
-
-var b1 = new Buffer('YW55=======', 'base64');
-var b2 = new Buffer('YW55', 'base64');
-
-assert.equal(
-  crypto.createHash('sha1').update(b1).digest('hex'),
-  crypto.createHash('sha1').update(b2).digest('hex')
-);
+if(common.hasCrypto) {
```

Comment:
styles:  Missing space before "(".  There are the same styles in this diff. 
Tests files are usually not checked by gjshint so we might need not to care about styles so much.